tuberculosis/lib/translated_app.dart

```diff
@@ -0,0 +1,162 @@
+import 'dart:async';
+
+import 'package:Tubuddy/api/api.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:Tubuddy/tubuddy_strings.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+final GlobalKey userSettings = GlobalKey();
+
+//class UserSettingsState extends State<UserSettingsWidget> {
```

Comment:
⚠️ COMMENTED CODE ⚠️
🚨 CODE SMELL DETECTED 🚨

Reply:
Zo'n prachtige GlobalKey is de flutter way. Zonder dat kan je geen aanpassingen doen omdat Flutter dan niet snapt dat alle dependencies van die widget opnieuw gemaakt moeten worden.